app/controllers/api/v1/cocktails_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V1::CocktailsController < ApplicationController
+    before_action :check_query_param
```

Comment:
Make sure that a consistent tab formatting is being used, it looks like the gemfile is a 2 space tab whereas this file is a 4 space tab. I think option+shift+f or alt+shift+f can be used for auto-formatting.

Reply:
Yes they were 4 spaces. Took it down to 2.